Rakefile

```diff
@@ -1,2 +1,7 @@
 require "bundler/gem_tasks"
 
+begin
+  require 'rspec/core/rake_task'
+    RSpec::Core::RakeTask.new(:spec)
+rescue LoadError
+end
```

Comment:
I am personally fine with leaving the spec running separate from the Rake tasks...